### ISSUE TITLE
Cannot render chinese page to png image.

### ISSUE THREAD

**Issue by wswuai** [scrapinghub/splash#267]
when i splash this url below(using web demo http://localhost:8050 ) , the splash returned image is full of unknown character

http://mp.weixin.qq.com/s?__biz=MzA4MjE0OTU3OA==&mid=207237261&idx=3&sn=be594e172aac972bbe4c254cd7a4d01b&3rd=MzA3MDU4NTYzMw==&scene=6#rd

i guess maybe render engine do not support other charset? (or do nothing abt other charset?)

the image is like that

![image](https://cloud.githubusercontent.com/assets/6711671/9105083/8aafe4e0-3c46-11e5-9d4d-5fe77494424e.png)

**Comment by wswuai** [comment]
if there is somewhere can config about charset 

please tell me and close this issue.

thanks a looooooooooot.

^o^

**Comment by kmike** [comment]
Hm, I see - when I try to render it using Splash locally (without Docker) it displays right characters, but in Docker it doesn't work. It looks like some fonts are missing in default Ubuntu 12.04 docker image, or Qt have troubles finding them.

**Comment by kmike** [comment]
https://github.com/scrapinghub/splash/commit/81c96dd25ffd71a2acc82c9166c0aed817fbbb34 fixes it. Thanks for the bug report!

To get use of this fix either wait for Splash 1.7 release (it will happen soon, likely next week) or install Splash master:

```
docker pull scrapinghub/splash:master
docker run -it -p 8050:8050 scrapinghub/splash:master
```

**Comment by wswuai** [comment]
Thank you for your help. cheeeeers!
